metabase/metabase
# Migrate slack forms to formik

Epic https://github.com/metabase/metabase/issues/26178

How to test:
- Go to Admin -> Settings -> Slack and complete the setup by following the instructions
- Make sure the slack status is displayed correctly afterwards

<img width="680" alt="Screenshot 2022-11-07 at 17 58 44" src="https://user-images.githubusercontent.com/8542534/200356355-f487bf50-a4d9-42c0-abde-52a14da6982f.png">
<img width="517" alt="Screenshot 2022-11-07 at 17 59 31" src="https://user-images.githubusercontent.com/8542534/200356367-85a46585-b97a-491a-868c-806828677733.png">
